docs/_component/editor.client.js

```diff
@@ -198,6 +211,46 @@ export function Editor({children}) {
                 <code>remark-math</code>
               </a>
             </label>
+            <label>
+              <input
+                checked={state.directive}
+                type="checkbox"
+                onChange={() =>
+                  setConfig({...state, directive: !state.directive})
+                }
+              />{' '}
+              Use{' '}
```

Comment:
Nit: For such constructs I think it’s nicer to use a single text node:

```suggestion
              />
              {' Use '}
```

Reply:
kind of agree yes, but I just copied the constructs above and didn't want to refactor too much things at once 🤪 

<img width="849" alt="CleanShot 2023-05-05 at 17 38 58@2x" src="https://user-images.githubusercontent.com/749374/236504096-67407d58-8dd9-4370-94dd-be8c668c68e5.png">

Reply:
Ah right, I didn’t notice. I think consistency is more important then.